docs/Over_OSPO-NL.md

```diff
@@ -1,7 +1,24 @@
-```{warning}
-De documentatie is nog volop in ontwikkeling
-```
-
 # Over OSPO-NL
- 
-TO DO
\ No newline at end of file
+
+OSPO staat voor Open Source Program Office. De toevoeging NL doelt op de Nederlandse samenwerking om kennis en best
+practices te delen voor het opzetten en uitvoeren van OSPO's (of OSPOs) bij organisaties in Nederland. Het is niet van

```

Comment:
Is het hier nog handig om een gebruikte tools / frameworks toe te voegen? 
Bv ORT en OSH voor inzicht ?

Reply:
Zeker! En nog veel meer ... maar niet in de 'over ons', denk ik? Ik heb deze PR dan ook al gemerged, zodat we nieuwe PRs kunnen maken met kleine wijzigingen ... anders zou deze te groot worden.

Overigens vind ik het wel een risico dat we hier informatie ergens anders vandaan opnieuw gaan publiceren. Ik zou liever naar de TODO Group en LF OSPO verwijzen dan content opnieuw publiceren ...